incubator/hnc/test/e2e/issues_test.go

```diff
@@ -209,4 +209,18 @@ var _ = Describe("Issues", func() {
 		// Expected: delete successfully
 		mustRun("kubectl delete subns", nsChild, "-n", nsParent)
 	})
+
+	It("Should create and delete an unpropagatable object - issue 771", func(){
+		// set up
+		mustRun("kubectl create ns", nsParent)
+		mustRun("kubectl create ns", nsChild)
+		mustRun("kubectl hns set", nsChild, "--parent", nsParent)
+		// Creating unpropagatable object; both namespaces should have a condition
+		mustRun("kubectl create rolebinding --clusterrole=cluster-admin --serviceaccount=default:default -n", nsParent, "foo")
+		// verify
+		runShouldContain("CannotPropagateObject", 1, "kubectl hns describe", nsParent)
```

Comment:
The comment says "both namespaces should have a condition" but you only appear to be testing one namespace for the condition? Ditto on line 224, you only appear to test that one namespace does *not* have a condition.

Reply:
Ah right, you should also add `runShouldContain` `CannotUpdateObject` condition in the child namespace.

Reply:
added test for child